vembrane/__init__.py

```diff
@@ -143,6 +152,17 @@ def main():
         help="Keep all annotations of a variant if at least one of them passes "
         "the expression.",
     )
+
+    def mapping(value: str) -> Dict[str, str]:
+        return dict(map(str.strip, v.strip().split("=")) for v in value.split(",") if v)
+
+    parser.add_argument(
+        "--rename",
+        default="",
+        type=mapping,
```

Comment:
In any case, the canonical way for mutliple arguments would rather be to define `nargs="+"` here, and thereby expect separation by whitespace.

Reply:
Either that or multiple `--rename` args. The problem with nargs="+" is that it doesn't work well if you have multiple options/arguments with nargs="+".